### ISSUE TITLE
Missing RAID arrays

### ISSUE THREAD

**Issue by Jacek-ghub** [MrPig91/PSChiaPlotter#102]
Hi, I have a RAID0 array that I wanted to use for my tmp folder.  However, ChiaPlotter is not seeing any RAID arrays.  I tried to check the code and potentially in function Get-ChiaMaxParallelCount when you use MSFT_Disk or MSFT_PhysicalDisk, those are unfortunately only returning physical drives, but not RAID arrays.  I tried to check MSoft docs for Get-CimInstance, but it looks like there is no way to query for RAID arrays, or most likely I missed it.

I did some searching, and one option would be to use MSFT_Volume (with Get-CimInstance), but that one doesn't indicate RAID type, just basic drive info.  

Another option would be to use diskpart, and check for type (maybe NOT "Partition", or rather for exact type - in my case "Stripe" / RAID0).  Here is a link to a PS script that could be used to jump start this part: https://gist.github.com/schakko/4713248

Sorry, this is the first time I see PS code, so this is where my understanding ends.

As mentioned, this is the first time I am using PS, but your utility is a fantastic tool!

**Comment by Sky1414** [comment]
Hello
Switch in  Basic and write directly the path
![image](https://user-images.githubusercontent.com/83701847/122014113-2483d800-cdbf-11eb-9870-b370ddca069f.png)

**Comment by Jacek-ghub** [comment]
Thank you @Sky1414.  I did that, and got "Exception calling ".ctor" with "1" arguments(s): ..." error message.  So, for whatever reason this is also no go for me.

I did try to rebuild the RAID array to use basic disks, but there was no option to do that.  The dynamic is selected by default, and looks like there is no way to change it (using Computer Management / Disk Management).

I also tried to use Storage Spaces, but that didn't work at all.  I couldn't create a pool, as selected NVMes were "not connected right" whatever it is.

So, at the moment, I cannot use Chia Plotter on that box (i9-10900, MSI Z490A Pro, Samsung 970 1TB NVMes).  Hopefully, the code can be modified to also recognize RAID arrays.

Thank you, Jacek

**Comment by Jacek-ghub** [comment]
I did map that RAID0 array, and looks like that works.  It shows in "Plotting and Farming Drives" as well as I could start a job using it.

Also, the info provided by "Get-CimInstance -ClassName Win32_MappedLogicalDisk" is about the same as the one given by diskpart (from the link in the first post) - although it has both size and free space, where diskpart only provides the size.

I also tried to use \\host\path format on the "Basic" section of directories, but ChiaPlotter is crashing with it.
